IntelliTect.TestTools.Console/ConsoleAssert.cs

```diff
@@ -251,37 +268,16 @@ public static string Execute(string givenInput, Action action, bool normalizeLin
             }
         }
 
-        private static string GetMessageText(string expectedOutput, string output)
+
+        private static string GetMessageText(string expectedOutput, string output, string equivalentOperatorErrorMessage=null)
```

Comment:
The default value for `equivalentOperatorErrorMessage` is not needed since it appears to be specified at all call sites.

```suggestion
        private static string GetMessageText(string expectedOutput, string output, string equivalentOperatorErrorMessage)
```